src/lib/menu/menu-directive.ts

```diff
@@ -139,19 +139,30 @@ export class MdMenu implements AfterContentInit, MdMenuPanel, OnDestroy {
   }
 
   /** Event emitted when the menu is closed. */
-  @Output() close = new EventEmitter<void | 'click' | 'keydown'>();
+  @Output() closed = new EventEmitter<void | 'click' | 'keydown'>();
+
+  /**
+   * Event emitted when the menu is closed.
+   * @deprecated Switch to `closed` instead
+   */
+  @Output() close = this.closed;
```

Comment:
Same question here as the select- should we just have one `openedChange`?
cc @kara @crisbeto

Reply:
That's fine by me, just need to mark it as a breaking change or deprecated.

Reply:
@jelbourn - Ok, to verify we want to change`open` and `close` to `openChange` with a signature like:

```
export interface MdMenuEvent {
   opened: boolean;
   type: void | 'click' | 'keydown'
}
```